storage-docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3.8"
```

Comment:
This version tag is no longer needed in docker compose https://forums.docker.com/t/docker-compose-yml-version-is-obsolete/141313/3